crates/node/src/cli/rpc.rs

```diff
@@ -67,36 +54,28 @@ impl FromStr for Cors {
 
 #[derive(Clone, Debug, clap::Args)]
 pub struct RpcParams {
-    /// Disable the RPC server.
-    #[arg(env = "MADARA_RPC_DISABLED", long, alias = "no-rpc")]
-    pub rpc_disabled: bool,
-
-    /// Listen to all network interfaces. This usually means that the RPC server will be accessible externally.
-    /// Please note that some endpoints should not be exposed to the outside world - by default, enabling remote access
-    /// will disable these endpoints. To re-enable them, use `--rpc-methods unsafe`
-    #[arg(env = "MADARA_RPC_EXTERNAL", long)]
+    /// Disables the user rpc endpoint. This includes all methods which are part
+    /// of the official starknet specs.
+    #[arg(env = "MADARA_RPC_DISABLE", long, default_value_t = false)]
+    pub rpc_disable: bool,
+
+    /// Exposes the user rpc endpoint on port `0.0.0.0`. This generally means
+    /// that rpc methods will be accessible from the outside world.
+    #[arg(env = "MADARA_RPC_EXTERNAL", long, default_value_t = false)]
     pub rpc_external: bool,
 
-    /// RPC methods to expose.
-    #[arg(
-		env = "MADARA_RPC_METHODS",
-		long,
-		value_name = "METHOD",
-		value_enum,
-		ignore_case = true,
-		default_value_t = RpcMethods::Auto,
-		verbatim_doc_comment
-	)]
-    pub rpc_methods: RpcMethods,
-
-    /// RPC rate limiting (calls/minute) for each connection.
-    ///
-    /// This is disabled by default.
-    ///
-    /// For example `--rpc-rate-limit 10` will maximum allow
-    /// 10 calls per minute per connection.
-    #[arg(env = "MADARA_RPC_RATE_LIMIT", long)]
-    pub rpc_rate_limit: Option<NonZeroU32>,
+    /// Enables the admin rpc endpoint. This includes additional rpc methods
+    /// which are not part of the official specs and can be used by node admins
+    /// to control their node at a distance. By default, this is exposed to
+    /// `localhost`.
+    #[arg(env = "MADARA_RPC_EXTERNAL", long, default_value_t = false)]
+    pub rpc_admin: bool,
+
+    /// Exposes the admin rpc endpoint on port `0.0.0.0`. This is especially
+    /// useful when running Madara from inside a docker container. Be very
+    /// careful however when exposing this endpoint to the outside world.
```

Comment:
Add elobration that this endpoint is secured and requires the user to add a proxy infront of it, etc. Essentially the note you have in the PR description should be summarised here for the user.